content/en/docs/aws/pipeline.md

```diff
@@ -173,6 +172,7 @@ In order for `ml-pipeline-ui` to read these artifacts:
            image: gcr.io/ml-pipeline/frontend:0.2.0
            name: ml-pipeline-ui
    ```
+1. By default, NodeInstanceRole has S3 permissions. If users are using customized worker node AMI, ensure that the role attached to the nodegroup running the ml-pipeline-ui pod has read access policy to s3.
```

Comment:
A few suggestions:

- Add an extra blank line before the paragraph
- Address the user with "you", rather than referring to them as "users" -> speaking directly to them can help
- Add an indefinite article "a" in "using a customized worker node AMI" and "a read access policy"
- Usually, "make sure"  > "ensure", because it's more formal
- "Node group" > "nodegroup" (unless it's `nodegroup`) - reference: AWS EKS docs https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html
- For variables, values and attribute names from the command line input/output and code, wrap text with backticks (`) "ml-pipeline-ui pod" -> "...`ml-pipeline-ui` pod"
- Capitalize "S" in "S3" (AWS)

```suggestion

1. By default, NodeInstanceRole has S3 permissions. If you are using a customized worker node AMI, make sure that the role attached to the node group running the `ml-pipeline-ui` pod has a read access policy to S3.
```

(or "read-access policy")